aries-site/src/data/structures/templates.js

```diff
@@ -267,6 +256,26 @@ export const templates = [
       'paginated lists',
     ],
   },
+  {
+    name: 'Matrix Navigation',
+    description:
+    'Matrix Navigation Guidance.',
+    available: true,
+    cardOrder: 100,
+    preview: {
+      background: 'background-front',
+      image: {
+        src: {
+          light: '/templateImages/MatrixNavigation.svg',
+          dark: '/templateImages/MatrixNavigation-invert.svg',
+        },
+        alt: 'HPE Lists Preview',
+        fit: 'contain',
+      },
+    },
+    relatedContent: [],
+    seoDescription: 'Matrix Navigation',
+  },
```

Comment:
We should update the description for the Navigation page three lines down.

Reply:
yes I forgot to add this when figma was updated